Malmo/src/PythonWrapper/CMakeLists.txt

```diff
@@ -27,11 +27,15 @@ set( OTHER_FILES
 
 python_add_module( MalmoPython SHARED ${CPP_SOURCES} )
 include_directories( MalmoPython ${PYTHON_INCLUDE_DIRS} )
-target_link_libraries( MalmoPython Malmo ${PYTHON_LIBRARIES} )
+target_link_libraries( MalmoPython Malmo )
```

Comment:
Maybe better to move line 30 into the else or just add ${PYTHON_LIBRARIES} on line 33. Otherwise saying add MalmoPython and Malmo twice.

Reply:
Fixed.